qlib/rl/order_execution/from_neutrader/config.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
```

Comment:
It looks like The code in `from_neutrader` was implemented to support specific examples.
Maybe we can move it to examples or tests.